source/client-side-encryption/tests/find.template.yml

```diff
@@ -0,0 +1,121 @@
+runOn:
```

Comment:
There needs to be a test of getMore.  You can do it easily with a find of four encrypted documents and a batchSize of 2.

Reply:
Added a passing getMore.template.yml test.